src/homepageExperience/components/steps/cli/InstallDependencies.tsx

```diff
@@ -0,0 +1,312 @@
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+import {
+  ComponentColor,
+  Button,
+  ButtonGroup,
+  Orientation,
+  Table,
+  BorderType,
+} from '@influxdata/clockface'
+
+import React, {useEffect, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {getBuckets} from 'src/buckets/actions/thunks'
+import {event} from 'src/cloud/utils/reporting'
+
+export const InstallDependencies = () => {
```

Comment:
```suggestion
export const InstallDependencies: FC = () => {
```
`FC` is a type exported by React. For example here: https://github.com/influxdata/ui/blob/master/src/App.tsx#L48